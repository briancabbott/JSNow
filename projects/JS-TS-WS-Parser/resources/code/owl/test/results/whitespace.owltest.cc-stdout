file (0 - 16)
  elem (1 - 2)
  elem (4 - 5)
  elem (10 - 11)
  elem (11 - 12)
  elem (15 - 16)
