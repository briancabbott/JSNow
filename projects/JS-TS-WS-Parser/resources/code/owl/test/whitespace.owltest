xbxxaxxxxxaaxxxb---
.whitespace 'x'
file = elem*
elem = 'a' | 'b'

