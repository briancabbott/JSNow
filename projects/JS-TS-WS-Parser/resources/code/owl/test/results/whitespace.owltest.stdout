. xb   xxa   xxxxxa    a   xxxb   
   elem  elem     elem elem   elem
